robustbench/model_zoo/cifar100.py

```diff
@@ -185,6 +186,21 @@ def forward(self, x):
 
         return torch.mean(torch.stack(out_list),dim=0)
 
+
+class Modas2021PRIME_RN18_100(ResNet):
```

Comment:
also for the same reason here `Modas2021PRIME_RN18_100` -> `Modas2021PRIMEResNet18` (and without `100` which, i guess, signifies the number of classes? but it's anyway the same for all CIFAR-100 models, so there is no need to mention it in the class name)